### PR TITLE
Remove block list hover effect from block list items that aren't links

### LINKED ISSUE #115: Block list shouldn't have hover effect if items are not links
It's confusing the affordance of the element

blockx

### PR DESCRIPTION
**Pull request by Stanton**
Also update code style.

Closes #115

### CODE REVIEW

File: stylesheets/_component.block-list.scss [hunk 1]
```diff
@@ -40,137 +40,119 @@
  */
 
 /* -------------------------------------------------------------------------- *\
-    Configuration
+	Configuration
```
Comment (langy) [inline stylesheets/_component.block-list.scss]: 4-space indentation has been replaced by hard-tabs

Comment (Stanton) [inline stylesheets/_component.block-list.scss]: Fixed

File: stylesheets/_component.block-list.scss [hunk 2]
```diff
@@ -207,5 +188,5 @@ $width: 500px;
 }
 
 .block-list--tall .block-list__item {
```
Comment (langy) [inline stylesheets/_component.block-list.scss]: Maybe nest the block-list__item inside the .block-list--tall so each object has its own line, as per lines 111, 167 etc.

**Comment by langy** [comment]
:+1:

File: stylesheets/_component.block-list.scss (hunk 2 above)
Comment (Stanton) [inline stylesheets/_component.block-list.scss]: Didn't think it was really needed for a single declaration

```
.block-list--tall .block-list__item {
    padding: $padding * 2;
}
```

**Pull request merged**